### PR TITLE
Add references to upgrade survey

### PR DESCRIPTION
**Pull request by therealmitchconnors**

### CODE REVIEW

File: content/en/docs/setup/upgrade/canary/index.md [hunk 1]
```diff
@@ -179,3 +179,7 @@ Make sure to use the `istioctl` version corresponding to the old control plane t
 old gateways and, to avoid downtime, make sure the old gateways are up and running before proceeding
 with the canary uninstall.
 {{< /tip >}}
+
+## Istio 1.9 Upgrade Survey
+
+If you have completed your upgrade to Istio 1.9, we would like to hear from you!  Please take a few minutes to respond to our brief [survey](https://forms.gle/JSPcDkF7NzkXsYRS8) to tell us our we're doing!
```
Comment (smawson) [inline content/en/docs/setup/upgrade/canary/index.md:185]:
```suggestion
If you have completed your upgrade to Istio 1.9, we would like to hear from you!  Please take a few minutes to respond to our brief [survey](https://forms.gle/JSPcDkF7NzkXsYRS8) to tell us how we're doing!
```

**Comment by craigbox** [comment]
Aside from the announcement, I might make these changes only on the 1.9 branch, as the survey is 1.9 specific?

I also recommend you change the last ! to a . in all cases.

Two notes on the form: it says "DRAFT", and the Slack link at the end only works if you're logged in (we can't craft a URL that deep-links you if you're not, unfortunately). You might like to call out the channel name?

**Comment by ericvn** [comment]
@therealmitchconnors It looks like this PR can be closed as there was a PR merged into the release-1.9 branch as Craig mentioned above.

**Pull request closed without merging**